repoarchivetool/app.py

```diff
@@ -8,12 +8,35 @@
 import authentication_interface
 
 archive_threshold_days = 30
-domain = "http://localhost:5000"
 organisation = "ONS-Innovation"
 
 app = flask.Flask(__name__)
 app.config['SECRET_KEY'] = os.urandom(24)
 
+
+def check_file_integrity(files_to_check: list):
```

Comment:
Try using a loop instead of checking for the specific filenames.  This way the code will be generic, shorter and can be re-used with different files without having to update the logic.

E.g:
```
def check_file_integrity(files: List[str], directory: str = './'):

for file in files:
    file_path = os.path.join(directory, file)
    if not os.path.isfile(file_path):
         storage_interface.get_bucket_content(file)

```
This way you can just call with a list of file names.  If the files are expected in a different directory to ./ then you can pass the directory variable in.